randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2974,6 +2979,8 @@ Extra - asset_id: collision_camera_036
   * Extra - actor_def: actordef:actors/props/breakablemag007/charclasses/breakablemag007.bmsad
   > Upper-right Corner
       Trivial
+  > Door to Central Unit
+      Trivial
```

Comment:
Just remove this connection to the Central Unit. It only needs to go to Upper-right Corner